docs/settings.md

```diff
@@ -19,9 +19,11 @@ equivalent keyword arguments, eg. `uvicorn.run("example:app", port=5000, reload=
 ## Development
 
 * `--reload` - Enable auto-reload.
-* `--reload-dir <path>` - Specify which directories to watch for python file changes. May be used multiple times. If unused, then by default all directories in current directory will be watched.
+* `--reload-dir <path>` - Specify which directories to watch for python file changes. May be used multiple times. If unused, then by default the whole current directory will be watched.
+* `--reload-include <glob-pattern>` - Specify a glob pattern to match files or directories which will be watched. May be used multiple times. By default the following patterns are included: `.py`. These can be overwritten by explicitly excluding them.
+* `--reload-exclude <glob-pattern>` - Specify a glob pattern to match files or directories which will excluded from watching. May be used multiple times. By default the following patterns are excluded: `.*, .py[cod], .sw.*, ~*`. These can be overwritten by explicitly including them.
```

Comment:
these are watchgod defaults as far as I know, in any case I'm not sure I'd enter that level of details in the docs unless there's a reason for it.

I recall we discussed at length this in another issue,

what happens in  the case no pattern is specified, which will be the most common use-case ?

Reply:
This is in the docs in order to show which files are ignored. These ignores can be explicitly overwritten with an include.
I could also use super(), but then there would be no way to override the defaults.

Reply:
@euri10  Should we remove this from the docs or do we want to keep it in there to let users know about these patterns?

Reply:
I like it to have it here as well as in the cli `--help` output

Reply:
Help text for include:
> Set glob patterns to include while watching
> for files. Includes '*.py' by default, which
> can be overridden in reload-excludes.

Help text for exclude:
>Set glob patterns to exclude while watching
>for files. Includes '.*, .py[cod], .sw.*,
>~*' by default, which can be overridden in
>reload-excludes.

Note I do not want to use the default parameter as it gives the impression that changing this will not include these patterns. Contrary to many other settings these patterns need to be explicitly negated.